checkov/common/output/csv.py

```diff
@@ -139,7 +138,7 @@ def persist_report_iac(self, file_name: str, output_path: str = "") -> None:
     def persist_report_container_images(self, file_name: str, is_api_key: bool, output_path: str = "") -> None:
         CSVSBOM.write_section(
             file=os.path.join(output_path, file_name),
-            header=HEADER_CONTAINER_IMAGE,
+            header=HEADER_OSS_PACKAGES,
```

Comment:
```suggestion
            header=HEADER_CONTAINER_IMAGE,
```
since they are the same anyway, please keep the original name, so no one is confused 😄

Reply:
Yeah left over from a period in which I deleted the HEADER_CONTAINER_IMAG
thanks!